### PR TITLE
Read _LIBCONTAINER_INITPIPE in nsexec.c

### PR DESCRIPTION
**Pull request by LK4D4**
This fixes ExtraFiles for execin
Thanks @Mic95 for report

### CODE REVIEW

**Comment by crosbymichael** [comment]
LGTM

File: integration/execin_test.go [hunk 1]
```diff
@@ -314,3 +314,78 @@ func TestExecInEnvironment(t *testing.T) {
...
+		ExtraFiles: []*os.File{pipein1, pipein2},
+		Stdin:      nil,
+		Stdout:     &stdout,
+	}
+	err = container.Start(inprocess)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	waitProcess(inprocess, t)
```
Comment (dqminh) [inline integration/execin_test.go]: should we wait for the original process (i.e., `cat`) here too ?

**Comment by mrunalp** [comment]
LGTM

File: integration/execin_test.go (hunk 1 above)
Comment (LK4D4) [inline integration/execin_test.go]: I think it is okay, it will be die on defer and not crucial for test.

**Comment by dqminh** [comment]
LGTM

**Pull request merged**